mne/forward/forward.py

```diff
@@ -392,6 +392,9 @@ def read_forward_solution(fname, force_fixed=False, surf_ori=False,
     surf_ori : bool, optional (default False)
         Use surface-based source coordinate system? Note that force_fixed=True
         implies surf_ori=True.
+    use_cps : bool
+        Whether to use cortical patch statistics to define normal
+        orientations.
```

Comment:
ok for me, `read_forward_solution` just calls `convert_forward_solution` at the end, so we could also take it out of the function